File: app/src/main/java/com/sergivonavi/materialbanner/app/activities/adapter/ItemViewHolder.java
package com.sergivonavi.materialbanner.app.activities.adapter;

import android.view.View;
import android.widget.ImageView;

import com.sergivonavi.materialbanner.app.R;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

class ItemViewHolder extends RecyclerView.ViewHolder {

    ImageView image;

    ItemViewHolder(@NonNull View itemView) {
        super(itemView);
        image = itemView.findViewById(R.id.item_image);
    }

}

File: app/src/main/java/com/sergivonavi/materialbanner/app/activities/adapter/ItemViewHolder.kt
package com.sergivonavi.materialbanner.app.activities.adapter

import android.view.View
import android.widget.ImageView
import androidx.recyclerview.widget.RecyclerView
import com.sergivonavi.materialbanner.app.R

internal class ItemViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    @JvmField
    var image: ImageView

    init {
        image = itemView.findViewById(R.id.item_image)
    }
}